src/dos/drive_local.cpp

```diff
@@ -246,7 +262,7 @@ bool localDrive::FindNext(DOS_DTA & dta) {
 		DOS_SetError(DOSERR_NO_MORE_FILES);
 		return false;
 	}
-	if(!WildFileCmp(dir_ent,srch_pattern)) goto again;
+	if (!WildFileCmp(dir_ent,srch_pattern)) goto again;
```

Comment:
Bleh; a loop implemented using `goto`. Out of scope of this review, but maybe a good candidate for a new issue for newcomers.

Reply:
Bleh is right.. this `goto` is followed by more code blocks each with their own branch points that can `goto again`.